prisma/schema.prisma

```diff
@@ -14,6 +14,7 @@ model User {
   email        String        @unique
   admin        Admin?
   hospitalRole HospitalRole[]
+  landingPage Landing_Page?
```

Comment:
I don't think the user needs a reference to the landing page. It should be the other way around right?

Reply:
what do you mean, how?